Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -117,12 +118,28 @@ public struct TableTopSceneView: View {
                     }
                 }
                 .onAppear {
-                    arViewProxy.session?.run(configuration)
+                    arViewProxy.session.run(configuration)
                 }
                 .onDisappear {
-                    arViewProxy.session?.pause()
+                    arViewProxy.session.pause()
                 }
             
+            if !coachingOverlayIsHidden {
+                ARCoachingOverlay(goal: .horizontalPlane)
+                    .sessionProvider(arViewProxy)
+                    .active(coachingOverlayIsActive)
+                    .allowsHitTesting(false)
+                    .overlay(alignment: .top) {
+                        if !helpText.isEmpty {
+                            Text(helpText)
+                                .frame(maxWidth: .infinity, alignment: .center)
+                                .padding(8)
+                                .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+                                .animation(.easeInOut, value: 1)
```

Comment:
this animation isn't working for me. I would think it should be this:
```suggestion
                                .transition(.opacity)
```

but that doesn't seem to be working either.

Reply:
[This works](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/461)